config/custom-environment-variables.json

```diff
@@ -11,6 +11,12 @@
     "useSSL": "USE_SSL"
   },
   "api": {
-    "url":  "TRIBUNALS_CASE_API_URL"
+    "url": "TRIBUNALS_CASE_API_URL"
+  },
+  "ssl": {
+    "hpkp": {
+     "maxAge": "TO_BE_DEFINED",
+     "sha256s": "TO_BE_DEFINED"
```

Comment:
This will need refer to encrypted value in vault

Reply:
Yep, to be clear node.js doesn't know anything about the vault. You need to define an env var in Azure and set it to the encrypted value from the vault. Once you know the name of both env vars I'll change `TO_BE_DEFINED ` to whatever you come up with.